# Arrived object
string object_name
geometry_msgs/Pose current_pose
---
